src/pages/HomePage.jsx

```diff
@@ -9,21 +14,56 @@ export default function CHRAT({messages, onNewMessage}) {
   }
   return (
     <>
-      <ChratBody>
-        <Header>
+      <StyledChratDiv>
+        {/* Header */}
```

Comment:
I would recommend to remove the comments in this return statement.

Reply:
all comments ? ok